src/platform/ESP32/CHIPPlatformConfig.h

```diff
@@ -75,3 +75,23 @@
 #else
 #define CHIP_DEVICE_CONFIG_ENABLE_WIFI_AP 0
 #endif
+
+#if CONFIG_ENABLE_ICD_SERVER
+
+#ifndef CHIP_CONFIG_ICD_IDLE_MODE_INTERVAL
```

Comment:
use _MS or _SEC or similar throught.

Reply:
I see these are defined on other platforms, so if this should not be part of the PR please create a separate issue, however the suffixes should be added.

Reply:
Added, the suffixes has been added in #29417